crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -7,7 +7,20 @@
 
 class ProjectController(Controller):
 
-    @route(["/together/project/<int:project_id>"], auth="public", website=True)
```

Comment:
Please restore this route path /together prefix for all routes in this module.

Reply:
I added the `/together` prefix, but is it necessary if together/crowdfunding is in the subdomain/domain ?

Reply:
What will be the final route then? Is it `crowdfunding.localhost:8069/together/project` or still `localhost:8069/together/project`? If the answer is the first, then simply change `crowdfunding` subdomain name by `together`and you can remove `/together` from the route. But could the routes conflict with other routes that would have the same path but in another subdomain?

Reply:
I think the best would be `together.compassion.ch/project`. I haven't tested but yes the same path on different websites/subdomains will probably conflict. If we need to have identical paths we can choose in the controller which template/page to render depending on the currently visited website/subdomain. Or use different routes as we don't have that many.